plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/storage/LibvirtStorageAdaptor.java

```diff
@@ -72,7 +72,12 @@
     private String _manageSnapshotPath;
 
     private String rbdTemplateSnapName = "cloudstack-base-snap";
-    private int rbdFeatures = (1 << 0); /* Feature 1<<0 means layering in RBD format 2 */
+    private final int RBD_FEATURE_LAYERING = 1;
```

Comment:
the errors with checkstuyle are due to the vars not being static (final). full capitalised is reserved for those constants only.